.github/workflows/release.yaml

```
@@ -90,25 +90,35 @@ jobs:
 
       - uses: ko-build/setup-ko@v0.6
         with:
-          version: v0.14.1
+          version: v0.15.1
+
+      - name: Install cosign
+        uses: sigstore/cosign-installer@v3.2.0
+        with:
+          cosign-release: "v2.2.2"
 
       - name: Prepare
         run: |
           echo "BUILD_DATE=$(date -u +'%Y-%m-%dT%H:%M:%SZ')" >> $GITHUB_ENV
+          echo "REPOS"="quay.io/grafana-operator/grafana-operator" "ghcr.io/${{ github.repository }}" >> $GITHUB_ENV
 
       - name: Build and push
         run: |
-          declare -a arr=("quay.io/grafana-operator/grafana-operator" "ghcr.io/${{ github.repository }}" )
-          
-          for i in "${arr[@]}"
+          for i in ${{ env.REPOS }}
           do
             export KO_DOCKER_REPO=${i}
-            ko build --sbom=none --bare --platform linux/arm64,linux/arm/v7,linux/amd64 -t ${{ github.ref_name }} \
+            ko build --sbom=spdx --image-refs ./image-digest-${i} --bare --platform linux/arm64,linux/arm/v7,linux/amd64 -t ${{ github.ref_name }} \
```

```suggestion
            ko build --sbom=spdx ./image-digest-${i%.*} --bare --platform linux/arm64,linux/arm/v7,linux/amd64 -t ${{ github.ref_name }} \
```

Suggesting this to drop all characters after the registry name, thus we'll now have something like `image-digest-quay` and `image-digest-ghcr`

The full repo name seems to be breaking ko build during my testing, which would return `No such file or directory` when attempting to pass the `--image-refs` path with a full repo path like `quay.io/hstefans/grafana-operator` I guess this is most likely because of the use of slashes, thus confusing the part of the logic responsible for writing to the specified path, as it sees `quay.io/ ` as a directory, rather than a filename.

IMHO we don't gain much by providing the full name anyway, and escaping `/` here is going to get ugly, so `${i%.*}` should be enough

---

Of course, easy and a good solution. Thanks for taking the time debugging this @HubertStefanski